src/Simulation/CsharpGeneration/FindNuspecReferences.ps1

```diff
@@ -59,13 +59,6 @@ function Add-PackageReferenceDependencies($ProjectFileName) {
 
 # Add dependencies for the projects included in this NuGet package.
 Add-PackageReferenceDependencies 'Microsoft.Quantum.CsharpGeneration.fsproj'
-Add-PackageReferenceDependencies '..\EntryPointDriver\EntryPointDriver.csproj'
```

Comment:
I am curious about why adding these references is no longer needed?

Reply:
This was added when I made EntryPointDriver part of the CsharpGeneration package, so that the package included dependencies for both projects. Since EntryPointDriver is no longer part of this package, the dependencies aren't needed anymore.